config/tech-docs.yml

```diff
@@ -2,7 +2,7 @@
 host: https://tdt-documentation.london.cloudapps.digital/
 
 # Header-related options
-show_govuk_logo: true
+show_govuk_logo: false
```

Comment:
Hey Ollie, why are we removing the logo (I'm sure you'll have a great reason 😄 ?

Reply:
`show_govuk_logo` was in this config file twice – once with `true`, once with `false` (which was later on line 42, and so over-wrote the first instance). This is just removing that duplication, and doesn't change the behaviour.

We hide the logo because the documentation for tech docs is not on a gov.uk domain, and so should not be using the crown or GOV.UK logotype.

Reply:
Great!